Sources/Orbit/Components/ListChoice.swift

```diff
@@ -0,0 +1,266 @@
+
+            // Provide minimal height (frame(minHeight:) collapses multiline text in snapshots)
+            Color.clear
+                .frame(width: 0, height: 48)
+
+            trailingElementView
+                .padding(.trailing, .medium)
+        }
+        .frame(maxWidth: Layout.readableMaxWidth, alignment: .leading)
+        .overlay(separator, alignment: .bottom)
+    }
+    
+    @ViewBuilder var header: some View {
+        if isHeaderEmpty == false {
+            Header(
+                title,
+                description: description,
+                iconContent: icon,
+                titleStyle: .title4,
+                descriptionSize: .small,
+                horizontalSpacing: .xSmall,
+                verticalSpacing: .xxxSmall
+            )
+            .padding(.vertical, headerTextPadding)
+            .padding(.leading, icon.isEmpty ? .medium : .small)
+            .padding(.trailing, .medium)
+        }
+    }
+
+    @ViewBuilder var trailingElementView: some View {
+        switch trailingElement {
+            case .none:
+                EmptyView()
+            case .disclosure(let color):
+                Icon(symbol: .chevronRight, size: .medium, color: color)
+            case .button(let type):
+                trailingButton(type: type)
+                    .padding(.vertical, .small)
+                    .disabled(true)
+            case .checkbox(let isChecked):
+                Checkbox(isChecked: isChecked)
+                    .disabled(true)
+        }
+    }
+    
+    @ViewBuilder func trailingButton(type: TrailingElement.ButtonType) -> some View {
+        switch type {
+            case .add:
+                Button(.plus, style: .primarySubtle, size: .small)
+            case .remove:
+                Button(.close, style: .criticalSubtle, size: .small)
+        }
+    }
+
+    var separator: some View {
+        HairlineSeparator()
+            .padding(.leading, separatorPadding)
+    }
+
+    var separatorPadding: CGFloat {
+        icon.isEmpty ? .medium : .xxLarge
+    }
+    
+    var isHeaderEmpty: Bool {
+        title.isEmpty && description.isEmpty && icon.isEmpty
+    }
+    
+    var headerTextPadding: CGFloat {
+        title.isEmpty || description.isEmpty ? .medium : .small
+    }
+}
+
+// MARK: - Inits
+public extension ListChoice {
+    
+    /// Creates Orbit ListChoice component.
+    init(
+        _ title: String = "",
+        description: String = "",
+        icon: Icon.Symbol = .none,
+        trailingElement: ListChoice.TrailingElement = .disclosure(),
+        backgroundColor: BackgroundColor? = nil,
+        action: @escaping () -> Void = {}
+    ) {
+        self.title = title
+        self.description = description
+        self.icon = .icon(icon, size: .default, color: .inkNormal)
+        self.trailingElement = trailingElement
+        self.backgroundColor = backgroundColor
+        self.action = action
+    }
+    
+    init(
```

Comment:
This init comment can mention the custom icon content (can be copied from Tile, Card etc.)